cron/auth02mozdef.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2016 Mozilla Corporation
+# Author: gdestuynder@mozilla.com
+
+# Imports auth0.com logs into MozDef
+
+import hjson
+import sys
+import requests
+import mozdef_client as mozdef
+try:
+    import urllib.parse
+    quote_url = urllib.parse.quote
+except ImportError:
+    #Well hello there python2 user!
+    import urllib
+    quote_url = urllib.quote
+
+class DotDict(dict):
```

Comment:
that would mean each of these feature potentially lives in their directory, such as:
cron/auth02mozdef/__init__.py
cron/auth02mozdef/DotDict.py
etc.

Not really against it though since these are currently self-contained programs its hard to say how it should be done (else i'd want a standard "shell" for the main code and run things as class/methods and have test cases)